April/30/jeff_offutt/info.yaml

```diff
@@ -0,0 +1,16 @@
+title: Jeff Offutt
+description: |
+    On April 30, 1961 was born Jeff Offutt.
+    He is an American professor of Software Engineering at George Mason University.
+    Offutt was editor-in-chief of Wiley’s journal of Software Testing, Verification and Reliability from 2007 through 2019, 
+    co-founded the IEEE International Conference on Software Testing, Verification and Validation (ICST), 
+    and was the first steering committee chair,
+    Offutt is known for many fundamental contributions to the field of software testing, 
+    in particular mutation testing, model-based testing, bypass testing of web applications,
+    and automatic test data generation.
+
+url: https://en.wikipedia.org/wiki/Jeff_Offutt
+images:
+    - filename: jeff_offutt.png
```

Comment:
Image is no good b/c it is distorted.

The original image is 432x640 px, so can't be scaled up to fit a 600x600 frame without losing quality or distorting it.

Take the original and increase the canvas width, filling in the remaining space with the background color (see README). At the same time crop the height to fit the required size.